frontend/src/app/pages/Data/Features/NationalMap/index.tsx

```diff
@@ -22,15 +23,12 @@ import Charts from './charts';
 /* Data Fetching */
 import { useSelector, useDispatch } from 'react-redux';
 import { useNationalMapSlice } from './slice';
-import { selectFacilities, selectLoading, selectError, selectFilters, selectCounties, selectBoundaries } from './slice/selectors';
+import { selectFacilities, selectLoading, selectError, selectFilters, selectCounties, selectBoundaries, selectStates } from './slice/selectors';
 import { Filters, FilterCompanies, Geography, CallType, FacilityType, SecureLVL } from './slice/types';
 import { Facility } from 'types/Facility'
-import { createImmutableStateInvariantMiddleware } from '@reduxjs/toolkit';
-import counties from 'us-atlas/counties-10m.json';
-import * as topojson from 'topojson-client';
 import yj from 'yieldable-json';
-import { from } from '@apollo/client';
 import { fifteenMinuteRate, maxCanonicalFacilityRate, stats } from './utils';
+import stateNormalizer from 'us-state-codes';
```

Comment:
Our db provides the state code (GA) while topoJson expects the full name. Added this package to normalize back and forth.